doc/api/zlib.markdown

```diff
@@ -109,6 +109,30 @@ http.createServer((request, response) => {
 }).listen(1337);
 ```
 
+By default, the zlib methods will throw an error when decompressing truncated
+data. If you know that your input may be incomplete, or you want to
+inspect only the beginning of a compressed file, you can change the flushing
+method which is used to compressed the last chunk of input data:
```

Comment:
Better :-) ... I'd like to avoid using `you`, however. Perhaps:

```
By default, the zlib methods with throw an error when decompressing truncated
data. However, if it is known that the data is incomplete, or the desire is to inspect
only the beginning of a compressed file, it is possible to suppress the default error handling by changing the flushing method that is used:
```